.tools/test/SPECIFICATION.md

```diff
@@ -0,0 +1,44 @@
+# Deployment Instructions
+These instructions assume a "target account" (where tests will be invoked) of "python" per [this repository's configuration](config/targets.yaml).
+You can replace Python with any of the other languages listed in this repository's configuration.
+
+To request an alternate configuration for your own repository, please [submit an issue](https://github.com/awsdocs/aws-doc-sdk-examples/issues/new?assignees=octocat&labels=type%2Fenhancement&labels=Tools&title=%5BEnhancement%5D%3A+Weathertop+Customization+Request&&) with the `Tools` label.
+
+## 1. Deploy Consumer Stack for your language (e.g. Python)
```

Comment:
These are instructions for end users to create the stack using Python CDK. We have discussed repeatedly that while Python was useful for your development, in our team's experience, Python CDK has not worked in a consistent cross platform manner. For that reason, we have agreed that we prefer CDK written in TypeScript. Do we need to re-open that conversation?

Reply:
Currently, all new projects using the CDK must use TypeScript.

These are modifications to legacy code that needs to be ported to TypeScript as soon as possible to comply with this team norm. 

This work will be done in https://github.com/awsdocs/aws-doc-sdk-examples/issues/5584.